domain/src/mongo.js

```diff
@@ -10,7 +10,14 @@ let db = undefined;
  * @returns {Promise.<Object, Error>} БД
  */
 async function connect(url) {
-    db = await mongo.connect(url);
+    do {
+        try {
+            db = await mongo.connect(url);
+        } catch (e) {
+            continue;
```

Comment:
use setTimeout or setInterval
your current solution is thread blocking and utilizes close to 100% cpu probably